packages/backend/src/database/migrations/20210728093219_saved_queries.ts

```diff
@@ -211,6 +211,7 @@ export async function up(knex: Knex): Promise<void> {
             .insert({
                 name: orgs[0].organization_name,
                 project_id: project.project_id,
+                is_private: false,
```

Comment:
We shouldn't change past migrations. At this point, there is no `is_private` column in the DB. 
The `.defaultTo(false);` in the new migration should be enough.

Reply:
yes, you are right, but I got an error here on `deploy` https://github.com/lightdash/lightdash/actions/runs/3428367523/jobs/5713447150

Reply:
oh yeah, I know why